src/app/shared/enums/events.ts

```diff
@@ -6,7 +6,10 @@ export enum UsageEventType {
   QUESTIONNAIRE_CANCELLED = 'QUESTIONNAIRE_CANCELLED',
   QR_SCANNED = 'qr_code_scanned',
   CLICK = 'click',
-  RESUMED = 'resumed'
+  RESUMED = 'resumed',
+  RECORDING_STARTED = 'recording_started',
+  RECORDING_STOPPED = 'recording_stopped',
+  RECORDING_ERROR = 'recording_error'
```

Comment:
Why some of the events are upper case and the others are lower case? Can we follow some convention here?

Reply:
Made the changes.